experimental/plutus-sample-project-ts/build.nix

```diff
@@ -0,0 +1,31 @@
+{ inputs, ... }:
+{
+  perSystem = { config, system, ... }:
+    let
+      mySchema = config.lbf-nix.lbfPlutusTypescript {
+        name = "myschema-lb";
+        src = ./lbf;
+        files = [ "MySchema.lbf" ];
+      };
```

Comment:
Move this to `lbf/build.nix`, make it a proper output in `packages.lbf-plutus-sample-project-ts`.

Also name the library `name = "lbf-plutus-sample-project-ts"`

Stick to the consistent naming throughout, as we do in other places, if you notice some incosistencies please note them down.

The idea is to have `lbf` libraries named per project, not per file in that project. Format: `lbf-<project-name>-<perhaps-additional-info>

Reply:
Thank you for the suggestion! I changed it to as you suggested!